aip/cloud/new.md

```diff
@@ -0,0 +1,123 @@
+powerful. They permit users to have very fine-grained permissions and delegated
+policy roles. They also allow a common policy definition (which might be quite
+complex) to be bound to numerous resoures.
+
+- **CreatePolicy** - Creates a shared policy resource
+- **GetPolicy** - Retrieves a shared policy resource
+- **UpdatePolicy** - Updates a shared policy resource
+- **DeletePolicy** - Deletes a shared policy resource
+- **ListPolicies** - Lists shared policy resources (typically for a given
+  `parent`)
+- **LookupEffectivePolicy** - Retrieves the effective policy for a given
+  `resource`, which is the combination of the polices bound to the resource and
+  any ancestors
+- **CreateBinding** - Creates a binding between a shared policy resource and a
+  specified resource
+- **GetBinding** - Retrieves a binding between a shared policy resource and a
+  resource
+- **DeleteBinding** - Deletes a binding between a shared policy resource and a
+  resource
+- **ListBindings** - Lists bindings between shared policy resources and other
+  resources. This method **must** accept either a reference to the shared
+  policy resource or the target resource (typically via the `filter` field),
+  and list all appropriate bindings.
+
+### Permissions
+
+There **should** be a separate permission for each policy method, per standard
+practice.
+
+Bindings are a bit special in that they connect two resources. Conceptually,
+it's easiest to think of a binding as two half-bindings, one connected to the
+Policy and one connected to the Resource. Each half-binding has it's own
+permission to create/delete. So to create a binding between a Policy and a
+Resource requires 2 permissions, but to break a binding only requires
+permission on either the Policy or the Resource.
+
+NOTE: At this point in time, no guidance is given for recreating a binding. In
+theory only half of a binding could be deleted, thus allowing only a
+single-permission to re-establish the missing half of the binding.
+
+## Effictive Policies
+
+An effective policy is the combination of policies applied to the ancestry of a
+resource. For example, to determine the effective policy for a VM would require
+loading the policies attacted to the VM, Project, Folder, and Organization.
+
+There are multiple combination stragies, and which one chosen depends on the
+Policy system. Generally speaking, policies **should** be combined in an
+additive fashion where leaf nodes take precedence for ALLOW semantics, and the
+root nodes taking precendence for DENY semantics. This strategy works well for
+large organizations where administrators need to delegate their responsibility
+and/or want to set reasonable defaults at the top with overrides being set
+where needed.
+
+In some use cases, a policy lower in the hierarchy taking precedency may be
+unacceptable. We strongly encourage such cases to permit configuration of
+override policies instead of forcing the choice.
```

Comment:
As a naive reader, I do not really understand what it is you are telling me to do here.

Reply:
I've rephrased this section. Please let me know if what I'm trying to communicate is clearer now.